src/app/space/[did]/root/[cid]/shard/[shard]/page.tsx

```diff
@@ -28,23 +29,38 @@ interface PageProps {
   }
 }
 
+const isCARLink = (link: Link): link is CARLink => link.code === CAR.codec.code
+
 export default function ItemPage ({ params }: PageProps): JSX.Element {
   const [{ client, spaces }] = useW3()
   const spaceDID = decodeURIComponent(params.did)
   const space = spaces.find(s => s.did() === spaceDID)
   const root = parseLink(params.cid)
-  const shard = parseLink(params.shard)
+  const shard = parseLink(params.shard).toV1()
```

Comment:
Does it need to be v1?

Reply:
Possibly not in theory, but `CARLink` is defined as v1 specifically. It's probably an accident, as `1` is the default for the type parameter. But it's either this or type guard for v1, and we may as well accept v0 and convert it, since it's both simpler to write and accepts more input.